airflow_metrics/airflow_metrics/patch_thread.py

```diff
@@ -65,7 +65,7 @@ def wrapped():
 @once
 def patch_thread():
     try:
-        if sys.argv[1] == 'scheduler':
+        if len(sys.argv) > 1 and sys.argv[1] == 'scheduler':
```

Comment:
Can we change this to `if sys.argv` instead of `len(sys.argv) > 1)`

Reply:
no that only guarantees `sys.argv[0]` exists